.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: xenial
+
+language: ruby
+
+git:
+  depth: 200
+
+rvm:
+  - 2.3.3
+
+before_install:
+  - sudo apt-get install -y libpango1.0-dev ghostscript
+  - gem install asciidoctor -v 1.5.8
+  - gem install coderay json-schema
```

Comment:
Do we want fixed versions there are well?

Reply:
I'm pretty flexible about this, but I'd lean towards using the latest packages as long as they are working fine.  Let me know what you prefer.

Reply:
I added the explicit versioning for coderay.  For json-schema, it actually seems that it isn't required, so I've removed it completely.